vowpalwabbit/ftrl.cc

```diff
@@ -357,10 +365,16 @@ template <bool audit>
 void learn_coin_betting(ftrl& a, base_learner& base, example& ec)
 {
   // update state based on the example and predict
-  coin_betting_predict(a, base, ec);
-  if (audit) GD::print_audit_features(*(a.all), ec);
-  // update state based on the prediction
-  coin_betting_update_after_prediction(a, ec);
+  if (coin_betting_predict(a, base, ec))
+  {
+    if (audit) GD::print_audit_features(*(a.all), ec);
+    // update state based on the prediction
+    coin_betting_update_after_prediction(a, ec);
+  }
+  else
+  {
+    logger::errlog_error("Your features have too much magnitude. Example is ignored");
```

Comment:
```suggestion
    logger::errlog_error("The features have too much magnitude. Example is ignored");
```

I think we should start avoiding using `You` in error logs where possible (see https://github.com/VowpalWabbit/vowpal_wabbit/pull/3529 trying to get error messages to be less agressive)

Reply:
Yeah makes sense.